### ISSUE TITLE
Inaccurate information on alt text and images

### ISSUE THREAD

**Issue by CBID2** [TheOdinProject/curriculum#27230]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Issue
The alt attribute subsection mentions that every image requires alt text. This is incorrect. Some web images are used for decorative purposes and [WC3](http://www.w3.org/TR/WCAG20-TECHS/H67.html) recommends to only use the null tag for these kinds of images. It's important for devs to know this distinction early so that they can make accessible content effectively.

## Suggestion
Add information on different types of images and how to describe them in an accessible yet effective manner.

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-links-and-images#alt-attribute

### (Optional) Discord Name

Chrissy

**Comment by JoshDevHub** [comment]
Thank you for submitting an issue.

I'm going to tag our @TheOdinProject/html-css team and get their thoughts on this.

**Comment by KevinMulhern** [comment]
We have this covered later on in the curriculum - https://www.theodinproject.com/lessons/node-path-advanced-html-and-css-meaningful-text#alternative-text

I think thats the probably the best place for it to maintain a good pace with the curriculum. This seems like more of an intermediate/advanced thing to know than something a student who's just learning about HTML for the first time should be concerned with.

**Comment by KevinMulhern** [comment]
Thanks @CBID2, would this be enough?
> Besides the src attribute, ~~every~~ most image elements must also have an alt (alternative text) attribute.

**Comment by CBID2** [comment]
> Thanks @CBID2, would this be enough?
> 
> > Besides the src attribute, ~~every~~ most image elements must also have an alt (alternative text) attribute.

That's a good start @KevinMulhern.

**Comment by KevinMulhern** [comment]
What else would we need to do @CBID2?

**Comment by CBID2** [comment]
> What else would we need to do @CBID2?

Here's what I came up with:
> Besides the src attribute, most image elements must also have an alt (alternative text) attribute. If your image is only used to make your website beautiful, for example, a background image, use the null tag.

**Comment by KevinMulhern** [comment]
That's ok, but we'll have to then explain what a null tag is - I think we might be overthinking this.

Is it absolutely necessary for someone to learn about this before they've even built their first site ([recipes](https://www.theodinproject.com/lessons/foundations-recipes))? do other resources cover this at the same time they are teaching basic html tags?

**Comment by CBID2** [comment]
> That's ok, but we'll have to then explain what a null tag is - I think we might be overthinking this.
> 
> 
> 
> Is it absolutely necessary for someone to learn about this before they've even built their first site ([recipes](https://www.theodinproject.com/lessons/foundations-recipes))? do other resources cover this at the same time they are teaching basic html tags?

The null tag is pretty simple (e.g.,"alt="") and it goes hand in hand with learning basic HTML tags. Plus, they'll learn that accessibility is not an afterthought, something that I haven't seen other coding curriculums do @KevinMulhern.

**Comment by KevinMulhern** [comment]
We agree accessibility is important, we have an [entire section just about it](https://www.theodinproject.com/paths/full-stack-javascript/courses/advanced-html-and-css#accessibility) later in the curriculum. We intentionally have it later to pace students through the material in a way we think makes most sense. Accessibility should absolutely be a concern, but only when our learners have had the chance and time to get the basics down. 

Thank you for taking the time to make this suggestion, but its not something we think will improve the curriculum.

**Comment by Jo-with-vision** [comment]
I appreciate it is closed now, so I apologize for an additional comment, but I can see that the issue was over-complicated and a simple solution could have been this:

"Besides the src attribute, every image element must also have an alt (alternative text) attribute.

The alt attribute is used to provide an alternative text description of an image. The text is used in place of an image if the image cannot be loaded, and used with assistive technologies, such as screen readers, to describe the image to visually impaired users. 

Although image elements must always include an alt attribute, the attribute does not always need to contain a text description (you will learn more about this when you cover accessibility principles later in the curriculum)." 

_note: i did do a little copy-editing sorry_

**Comment by CBID2** [comment]
> I appreciate it is closed now, so I apologize for an additional comment, but I can see that the issue was over-complicated and a simple solution could have been this:
> 
> 
> 
> "Besides the src attribute, every image element must also have an alt (alternative text) attribute.
> 
> 
> 
> The alt attribute is used to provide an alternative text description of an image. The text is used in place of an image if the image cannot be loaded, and used with assistive technologies, such as screen readers, to describe the image to visually impaired users. 
> 
> 
> 
> Although image elements must always include an alt attribute, the attribute does not always need to contain a text description (you will learn more about this when you cover accessibility principles later in the curriculum)." 
> 
> 
> 
> _note: i did do a little copy-editing sorry_
> 
> 

Aww that's exactly how I wanted to word it
